src/Resources/Wacc/Gifts/Gifts.ts

```diff
@@ -35,4 +35,9 @@ export default class Gifts extends AbstractResource {
         method: 'GET',
         path: '/{id}/print',
     });
+
+    public picking: Function = this.createMethodFromPartialSpec({
```

Comment:
`picking` doesn't seem like a verb like the rest of these methods. What does it mean, and is there a more verby way it can be expressed?

Reply:
I've set the resource as `getForPicking` with path `/pick`, hope that will work fine!